packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtons.js

```diff
@@ -14,7 +14,10 @@ export default function TaskNavButtons({
   const [saving, setSaving] = useState(false)
 
   return (
-    <Box direction='row'>
+    <Box
+      direction='row'
+      justify='center'
```

Comment:
What project was this needed for? The buttons in this component grow to their container, so why is `justify` added?

Reply:
I think this is needed for survey task workflows per these changes, because the viewer and task area change from horizontal to vertical at a wider screen width than previously. 

With a survey task, and a screen width of less than 1120px (70rem) per these changes - the task area drops below the viewer (vertical layout) and the task area is wide enough that the PrimaryButtons (Back, Done, etc.) hit their [max-width](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-react-components/src/PrimaryButton/themes/blue.js#L30). The container div aligns the buttons at the start without the `justify: center`.

Without `justify: center`:
![survey_task_area_without_justify_center](https://github.com/user-attachments/assets/5f713b6d-4f31-4b8b-921c-42e20687edf3)

With `justify: center`:
![survey_task_area_with_justify_center](https://github.com/user-attachments/assets/4e4f4d6c-a06e-43ec-a8d1-a1dfd2ef365e)